corehq/apps/reports/standard/deployments.py

```diff
@@ -319,3 +323,82 @@ def _bootstrap_class(obj, severe, warn):
         return "label label-warning"
     else:
         return "label label-success"
+
+
+class ApplicationErrorReport(GenericTabularReport, ProjectReport):
+    name = ugettext_noop("Application Error Report")
```

Comment:
Ah interesting. Looks like `rendered_report_title()` does what ugettext_lazy is designed for. I wonder if its behavior differs... anyways, reconciling them would be out of scope for this PR.